src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -141,6 +157,120 @@ public void traverse() {
         }
     }
 
+    @DataProvider
+    public Object[][] sequenceDictionaryTestValues() {
+
+        final String miniDictFileName = ReferenceUtils.getFastaDictionaryFileName(hg19MiniReference);
+        final String smallDictFileName = ReferenceUtils.getFastaDictionaryFileName(v37_chr17_1Mb_Reference);
+        final String largeDictFileName = ReferenceUtils.getFastaDictionaryFileName(b37_reference_20_21);
```

Comment:
Include the reference name in these variable names for clarity's sake later on (eg., "hg19MiniDictFile", "b37_20_21DictFile", "v37_chr17DictFile", etc.) You can safely auto-rename in IntelliJ using Refactor -> Rename.